docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+
+### Install Helm
+
+We will use Helm to install etcd Operator, Service Catalog and OSBA. In order to install Helm in your cluster, use the `helm` CLI:
+
+```console
+helm init
+```
+
+### Install etcd
+
+#### Create Storage Account
+
+In order to configure etcd operator to use Azure Blob Storage for backup and recovery purposes, you will need to first create a storage account:
+
+
+First, save your desired account name to an environment variable. It will be reused a few times.
+ **Bash**
+ ```console
+export AZURE_STORAGE_ACCOUNT=<STORAGE ACCOUNT>
+ ```
+
+**PowerShell**
+```console
+$env:AZURE_STORAGE_ACCOUNT = "<STORAGE ACCOUNT>"
+```
+
+Now use the az cli to create a new storage account.
+
+**Bash**
+```console
+az storage account create -n $AZURE_STORAGE_ACCOUNT -g aks-group
+```
+
+**PowerShell**
+```console
+az storage account create -n $AZURE_STORAGE_ACCOUNT -g aks-group
+```
+
+Once the account has been created, retrieve the keys.
+
+**Bash**
+```console
+az storage account keys list -n $AZURE_STORAGE_ACCOUNT -g aks-group -o table
+```
+
+**Powershell**
+```console
+az storage account keys list -n $env:AZURE_STORAGE_ACCOUNT -g aks-group -o table
+```
+
+Save the storage account key to an environment variable for later use.
+ **Bash**
+ ```console
+export AZURE_STORAGE_KEY=<STORAGE_KEY>
+ ```
+
+**PowerShell**
+```console
+$env:AZURE_STORAGE_KEY = "<STORAGE_KEY>"
+```
+
+Next, you will need to create a container for backup storage.
+
+**Bash**
+```console
+az storage container create --name etcd-backups --account-name $AZURE_STORAGE_ACCOUNT --account-key $AZURE_STORAGE_KEY
+```
+
+**PowerShell**
+```console
+az storage container create --name etcd-backups --account-name $env:AZURE_STORAGE_ACCOUNT --account-key $env:AZURE_STORAGE_KEY
+```
+
+#### Install etcd Operator
+
+Service Catalog requires etcd to persist state for everything it manages. We recommend using etcd Operator to install and manage an etcd cluster for Service Catalog. We have included sample values.yaml file to setup etcd operator with a version that supports Azure backup. 
+
+```console
+helm install --name etcd-operator stable/etcd-operator --values=contrib/k8s/etcd-operator/etcd-operator-values.yaml
+```
+
+This will create three deployments: etcd-operator, etcd-backup-operator, and restore-operator. This will also create three new Custom Resource Definitions: `etcdclusters.etcd.database.coreos.com`, `etcdbackups.etcd.database.coreos.com` and `etcdrestores.etcd.database.coreos.com`. These can be used to create an etcd cluster, along with backup and restore operations. 
+
+#### Create etcd Cluster
+
+Once etcd Operator has been installed, you can create a cluster. For production scenarios, we recommend a three node cluster. For convenience purposes, we have provided a Helm chart that will create a cluster and configure a Kubernetes CronJob to enable automatic backups of the cluster.
```

Comment:
it would help to say why a 3 node cluste is good, like "blah blah HA blerp bleep"

Reply:
~For convenience purposes~